docs/installation/mailer.md

```diff
@@ -17,7 +17,23 @@ values:
 ]
 ```
 
-Actually, the only thing required is the `fromEmail` value. If you look at the code of `Da\User\Factory\MailFactory.php` 
+Actually, the only thing required is the `fromEmail` value. 
+May be we need to set this is equal to `supportEmail` from your `params` configs (like yii2-app-advanced template):
```

Comment:
I'd write:

*If you want to set it the same as supportEmail from your config params (like yii2-app-advanced template):*

Reply:
Engrish 🤣